<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>translatorGUI</class>
 <widget class="QWidget" name="translatorGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>translatorGUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_lang1"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_lang2"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Dictionnary</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidget_dic1"/>
      </item>
      <item row="0" column="1">
       <widget class="QListWidget" name="listWidget_dic2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>DataBase</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_base1Add"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_base2Add"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_add">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_base1Del"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_base2Del"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_del">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>lineEdit_base1Add</zorder>
     <zorder>lineEdit_base2Add</zorder>
     <zorder>lineEdit_base2Add</zorder>
     <zorder>pushButton_add</zorder>
     <zorder>label_base1</zorder>
     <zorder>label_base2</zorder>
     <zorder>layoutWidget</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
